script.js

```diff
@@ -0,0 +1,140 @@
+const name = document.querySelector("input[name=fullname]");
+name.addEventListener("blur", (event) => {
+  checkName(event);
+});
+
+function checkName(event) {
+  let nameWords = name.value.split(" ").filter(function (el) {
+      return el != null && el != "" && el != undefined;
+    });
+
+  if(nameWords.length >= 2) {
```

Comment:
3 must be maxed length
`... the name provided by the user consists of exactly 2 or 3 words ...`